cypress/page/personal-form.page.ts

```diff
@@ -102,4 +66,17 @@ class PersonalFormPage {
     }
 }
 
+namespace PersonalFormPage {
+    export interface PersonalInformation {
+        name: string;
+        lastName: string;
+        email: string;
+        gender: string;
+        dateOfBirth: string;
+        mobileNumber: string;
+        hobbies: string[];
+        currentAddress: string;
+    }
```

Comment:
just the interface at the beginning of the file without the namespace is enough to this case :)